test/integration/services/data/test-get-auto-approval-config.js

```diff
@@ -1,44 +1,33 @@
 var expect = require('chai').expect
 var getAutoApprovalConfig = require('../../../../app/services/data/get-auto-approval-config')
-var config = require('../../../../knexfile').asyncworker
-var knex = require('knex')(config)
+var knexConfig = require('../../../../knexfile').asyncworker
+var knex = require('knex')(knexConfig)
+var config = require('../../../../config')
 var moment = require('moment')
 
 describe('services/data/get-auto-approval-config', function () {
   var insertedIds
 
-  before(function () {
-    return knex('IntSchema.AutoApprovalConfig')
-      .insert([{
-        Caseworker: 'caseworker1@test.com',
-        DateCreated: moment().toDate(),
-        AutoApprovalEnabled: 'true',
-        CostVariancePercentage: '5.00',
-        MaxClaimTotal: '100.00',
-        MaxDaysAfterAPVUVisit: '28',
-        MaxNumberOfClaimsPerYear: '10',
-        RulesDisabled: 'auto-approval-rule-1,auto-approval-rule-2,auto-approval-rule-3',
-        IsEnabled: 'true'
-      },
-      {
-        Caseworker: 'caseworker2@test.com',
-        DateCreated: moment().subtract(1, 'day').toDate(),
-        AutoApprovalEnabled: 'true',
-        CostVariancePercentage: '5.00',
-        MaxClaimTotal: '100.00',
-        MaxDaysAfterAPVUVisit: '28',
-        MaxNumberOfClaimsPerYear: '10',
-        RulesDisabled: 'auto-approval-rule-1,auto-approval-rule-3',
-        IsEnabled: 'false'
-      }])
-      .returning('AutoApprovalConfigId')
+  it('should return auto approval config defaults when no auto approval config data is found', function () {
```

Comment:
This is failing for me as I applied migration to existing DB which already had the default auto-approval row added (which will happen in CI).

```
services/data/get-auto-approval-config should return auto approval config defaults when no auto approval config data is found:
     AssertionError: expected null to equal 4
```